ci-operator/step-registry/aws-neuron-operator/rosa-cluster-provision/aws-neuron-operator-rosa-cluster-provision-commands.sh

```
@@ -68,6 +68,18 @@ fi
 AWS_ACCOUNT_ID=$(rosa whoami --output json | jq -r '."AWS Account ID"')
 AWS_ACCOUNT_ID_MASK=$(echo "${AWS_ACCOUNT_ID:0:4}***")
 
+# Billing account (required for HCP clusters)
+BILLING_ACCOUNT=""
+BILLING_ACCOUNT_MASK=""
+if [[ -f "${CLUSTER_PROFILE_DIR}/aws_billing_account" ]]; then
+  BILLING_ACCOUNT=$(head -n 1 "${CLUSTER_PROFILE_DIR}/aws_billing_account")
+  BILLING_ACCOUNT_MASK=$(echo "${BILLING_ACCOUNT:0:4}***")
```

Shellcheck (0.11.0)

[style] Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.

(SC2116)